api/versions.go

```diff
@@ -391,6 +392,17 @@ func (self *version2_0) HandleRequest(requestType string, request []string, m ma
 		return writeResult(fi, w)
 	case eventsApi:
 		return handleEventRequest(request, m, w, r)
+	case psApi:
```

Comment:
Also add to SupportedRequestTypes(). Realized we can make this interface better and less manual :P